docs/getting-started/install-crossplane.md

```diff
@@ -0,0 +1,83 @@
+---
+toc_hide: true
```

Comment:
This is only intended to be an included page. This means it doesn't need a title to display (since it won't be a standalone page) and it uses the `toc_hide` frontmatter to keep it out of the left-side table of contents.

Reply:
Maybe mark it [headless](https://gohugo.io/content-management/page-bundles/#headless-bundle)?

Reply:
This is covered in the docs contributing guide:
https://github.com/crossplane/crossplane/blob/master/docs/contributing/docs.md?plain=1#L97